### ISSUE TITLE
Code Fix for NUnit2010 on Ref Structs Creates CS1503

### ISSUE THREAD

**Issue by JasonBock** [nunit/nunit.analyzers#475]
This might be similar to an issue [I just submitted](https://github.com/nunit/nunit.analyzers/issues/474)

Write this test:

```csharp
[Test]
public static void EqualityWithSpans()
{
  var span1 = new[] { 1 }.AsSpan();
  var span2 = new[] { 1 }.AsSpan();

  Assert.That(span1 == span2, Is.True);
}
```

This triggers NUnit2010. The code fix creates this code:

```csharp
[Test]
public static void EqualityWithSpans()
{
  var span1 = new[] { 1 }.AsSpan();
  var span2 = new[] { 1 }.AsSpan();

  Assert.That(span1, Is.EqualTo(span2));
}
```

But then this raises the CS1503 compiler error.

**Comment by manfred-brands** [comment]
This will affect also other operator constraints like: `Assert.That(comparableRef1 < comparableRef2, Is.True);`
